include/deal.II/non_matching/fe_values.h

```diff
@@ -223,7 +223,27 @@ namespace NonMatching
     template <bool level_dof_access>
     void
     reinit(
-      const TriaIterator<DoFCellAccessor<dim, dim, level_dof_access>> &cell);
+      const TriaIterator<DoFCellAccessor<dim, dim, level_dof_access>> &cell,
+      const unsigned int q_index       = numbers::invalid_unsigned_int,
+      const unsigned int mapping_index = numbers::invalid_unsigned_int);
```

Comment:
`q_index` and `mapping_index` do the same thing here as in `hp::FEValues`, right? Could you either cite that class or add some documentation for those two arguments?